core/src/main/resources/defaults/blueprints/cdp-sdx.bp

```diff
@@ -259,101 +295,53 @@
             "value": "1"
           }
         ],
+        "serviceType": "KAFKA"
+      },
+      {
+        "refName": "knox",
+        "serviceType": "KNOX",
         "roleConfigGroups": [
           {
-            "refName": "kafka-GATEWAY-BASE",
-            "roleType": "GATEWAY",
-            "base": true
-          },
-          {
-            "refName": "kafka-KAFKA_BROKER-BASE",
-            "roleType": "KAFKA_BROKER",
+            "refName": "knox-KNOX_GATEWAY-BASE",
+            "roleType": "KNOX_GATEWAY",
+            "configs": [
+              {
+                "name": "ldap_connection_password",
+                "value": "admin123!"
```

Comment:
Again, should not be hardwired.

Reply:
According to #5143, all Knox LDAP properties were already generated by the Java code I mentioned, but are now getting removed. Knox will be using PAM to authenticate, so this and all related properties below need to be removed and not re-added at all (as they have become obsolete).

Reply:
will remove the LDAP configurations.